clippy_lints/src/utils/conf.rs

```diff
@@ -359,7 +359,11 @@ define_Conf! {
     /// Lint: ENUM_VARIANT_NAMES.
     ///
     /// The minimum number of enum variants for the lints about variant names to trigger
-    (enum_variant_name_threshold: u64 = 3),
+    (enum_name_threshold: u64 = 3),
```

Comment:
The config variable from the other lint is still different now. This should probably be changed back to `enum_variant_name_threshold` (else we'd need the `#[conf_deprecated]`, and avoiding it is why we changed back in the first place, right?).

Reply:
my bad, I forgot the variant part of the name, I should have look at the PR diff